pages/dkp/konvoy/2.1/choose-infrastructure/aws/air-gapped/seed-a-registry/index.md

```diff
@@ -14,7 +14,7 @@ Before creating a Kubernetes cluster you must have the required images in a loca
 1.  Download the images bundle.
 
     ```bash
-    curl -o konvoy-image-bundle.tar.gz -O http://downloads.d2iq.com/konvoy/airgapped/v2.1.1/konvoy_image_bundle_v2.1.1_linux_amd64.tar.gz
+    curl -o konvoy-image-bundle.tar.gz -O -L https://downloads.d2iq.com/konvoy/airgapped/v2.1.1/konvoy_image_bundle_v2.1.1_linux_amd64.tar.gz
```

Comment:
These have both `-o` and `-O` specified - please remove one or the other as they will conflict.

Reply:
Lulz, yea its super inconsistant, I'll go through and audit these pages on all the curl commands, and also switch everything over to the long options so its easy to see what those options do.